docs/dev/development.md

```diff
@@ -1,70 +1,90 @@
 # Local Development Setup
-To use this development environment on OSX, you need to have Docker and Boot2docker installed.
+To use this development environment on OSX, you need to have Java, Docker and VirtualBox installed (Boot2docker is optional).
```

Comment:
A link would be nice about how to get such environment.

Reply:
I am afraid people will download the Docker for Mac (https://docs.docker.com/docker-for-mac/) and with that Cloudbreak does not work